Examples/SDWebImage Demo/DetailViewController.m

```diff
@@ -24,6 +24,8 @@ - (void)configureView {
     [self.imageView sd_setImageWithURL:self.imageURL
                       placeholderImage:nil
                                options:SDWebImageProgressiveLoad];
+    self.imageView.shouldCustomLoopCount = YES;
+    self.imageView.animationRepeatCount = NSIntegerMax;
```

Comment:
If you want infinity loop, actually, use 0 is OK :)

Reply:
OOKK, I just missed it xdddddd

Reply:
> If you want infinity loop, actually, use 0 is OK :)

OOKK, I just missed it xdddddd